ERROR - 2022-03-27 13:59:50 --> Error connecting to the database: Access denied for user '****'@'localhost' (using password: YES)
CRITICAL - 2022-03-27 13:59:50 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
#0 /home/adry2296/public_html/ecommerceputra_core/vendor/codeigniter4/framework/system/Database/BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 /home/adry2296/public_html/ecommerceputra_core/vendor/codeigniter4/framework/system/Database/BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query()
#2 /home/adry2296/public_html/ecommerceputra_core/vendor/codeigniter4/framework/system/Model.php(728): CodeIgniter\Database\BaseBuilder->get()
#3 /home/adry2296/public_html/ecommerceputra_core/app/Models/CategoriesModel.php(17): CodeIgniter\Model->__call()
#4 /home/adry2296/public_html/ecommerceputra_core/app/Controllers/Frontend/HomeController.php(20): App\Models\CategoriesModel->getCategories()
#5 /home/adry2296/public_html/ecommerceputra_core/vendor/codeigniter4/framework/system/CodeIgniter.php(825): App\Controllers\Frontend\HomeController->index()
#6 /home/adry2296/public_html/ecommerceputra_core/vendor/codeigniter4/framework/system/CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController()
#7 /home/adry2296/public_html/ecommerceputra_core/vendor/codeigniter4/framework/system/CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest()
#8 /home/adry2296/public_html/ecommerceputra_public/index.php(37): CodeIgniter\CodeIgniter->run()
#9 {main}
